plugins/cost-insights/src/api/CostInsightsApi.ts

```diff
@@ -138,7 +138,7 @@ export type CostInsightsApi = {
    *
    * @param options Options to use when fetching insights for a particular cloud product and interval timeframe.
    */
-  getProductInsights(options: ProductInsightsOptions): Promise<ProductCost>;
+  getProductInsights(options: ProductInsightsOptions): Promise<Entity>;
```

Comment:
there will be large numbers of references to the `@backstage/catalog-model` version of `Entity` throughout all backstage code - no biggie, but just noting that introducing a bare type name `Entity` may be confusing.

Reply:
Ah, thanks for the heads up. I think we can come up with something a bit more descriptive.